.rubocop.yml

```diff
@@ -43,4 +43,10 @@ Metrics/BlockLength:
   Exclude:
     - 'config/routes.rb'
     - 'spec/**/*'
-    - 'config/environments/*'
\ No newline at end of file
+    - 'config/environments/*'
+
+Metrics/ClassLength:
+  Description: 'Avoid classes longer than 110 lines of code.'
+  Enabled: true
+  CountComments: false
+  Max: 110
```

Comment:
It is a bad to modify rubocop cops. Better to exclude file from it. I already done it in my last commits. =)